src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -335,7 +337,7 @@ static boolean isValidSnowflakeTableName(String tableName) {
    * @return connector name
    */
   static String validateConfig(Map<String, String> config) {
-    boolean configIsValid = true; // verify all config
+    Set<String> invalidConfigParams = new HashSet<>(); // verify all config and list invalid ones
```

Comment:
Instead of a set of String, lets create an Immutable class may be called 'InvalidParameters` with few private final strings called configName, configValue and then errorMsg. 

This will reduce a lot of redundant messaging here starting with LOGGER.error(Loggin.logMessage)

That immutable class can have a function called logInvalidProperties which can accept logger

Reply:
WDYT?

Reply:
That was my original intention, but I wasn't sure if it was too much for this logging. I'll make the change!